server/routes/serviceProviderReferrals/dashboardPresenter.ts

```diff
@@ -1,42 +1,47 @@
 import { SentReferral, ServiceCategory } from '../../services/interventionsService'
 import CalendarDay from '../../utils/calendarDay'
-import PresenterUtils from '../../utils/presenterUtils'
+import PresenterUtils, { SortableTableHeaders, SortableTableRow } from '../../utils/presenterUtils'
 import utils from '../../utils/utils'
 
 export default class DashboardPresenter {
   constructor(private readonly referrals: SentReferral[], private readonly serviceCategories: ServiceCategory[]) {}
 
-  readonly tableHeadings = ['Date received', 'Referral', 'Service user', 'Intervention type', 'Caseworker', 'Action']
+  readonly tableHeadings: SortableTableHeaders = [
+    { text: 'Date received', sort: 'none' },
```

Comment:
What does the new `sort` field do here, and why didn't we have one before?

Reply:
previously all the headers had 'sort: none' hardcoded in the template. this allows them to be altered by the presenter. it sets the default sort for the table.